perun-base/src/test/resources/sponsored-accounts-config.yml

```diff
@@ -0,0 +1,8 @@
+---
+namespaces:
+
+  dummy_namespace:
+    attributes_rules:
+      login: REQUIRED
+      password: OPTIONAL
+...
```

Comment:
I think there should be a new line at the end of the file.

Reply:
Fixed